gulpfile.js

```diff
@@ -50,32 +51,35 @@ gulp.task('compute-hashes', ['package'], function() {
 });
 
 gulp.task('deploy-vsix', ['package', 'compute-hashes'], function() {
-  if (process.env.TRAVIS_BRANCH != 'master') {
+  if (process.env.TRAVIS_BRANCH !== 'master') {
```

Comment:
Are you sure you don't need one or two other `=` signs ? We never know, it's JS... :trollface:

Reply:
:man_facepalming: